src/Admin/AbstractAdmin.php

```diff
@@ -2726,7 +2726,7 @@ public function getDashboardActions()
     final public function showMosaicButton($isShown)
     {
         if ($isShown) {
-            $this->listModes['mosaic'] = ['class' => self::MOSAIC_ICON_CLASS];
+            $this->listModes['mosaic'] = ['class' => static::MOSAIC_ICON_CLASS];
```

Comment:
maybe here too to be consistent?
https://github.com/sonata-project/SonataAdminBundle/blob/56b0201c63c50e61e6bb002c78fdd1991924d347/src/Admin/AbstractAdmin.php#L452

Reply:
You can't use static when defining a default value for a property.

Reply:
what property? I don't understand what you mean

Reply:
```
    protected $listModes = [
        'list' => [
            'class' => 'fa fa-list fa-fw',
        ],
        'mosaic' => [
            'class' => self::MOSAIC_ICON_CLASS,
        ],
    ];
```
https://3v4l.org/shiFi